### DISCUSSION TITLE
How to stop using the socket/Pangolin viewer and just use Rviz (SLAM requires a viewer connection but I dont want this)

### DISCUSSION THREAD

**Discussion by AmineDh98** [stella-cv/stella_vslam#525] [Q&A] [answered]
I am using ROS in my project and I prefer to use Rviz for my data visualization. When I run the stella_vslam_ros node it requires a socket connection and I have to connect the app.js of the socket viewer, otherwise it keeps trying to connect until it returns an error.
Do you have any idea how to break this socket/Pangolin viewer connection with the slamin order to just publish the data in a ros topic ?

**Comment by zoldaten** [comment, marked as answer]
ros2 run stella_vslam_ros run_slam --disable-mapping -v ~/lib/stella_vslam_examples/build/orb_vocab.fbow \
    -c ~/lib/stella_vslam/example/aist/equirectangular.yaml **--viewer none**
